spec/features/work_packages/new_work_package_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'spec_helper'
+require 'features/work_packages/details/inplace_editor/work_package_field'
+require 'features/work_packages/work_packages_page'
+
+describe 'new work package', js: true, retry: 3, retry_wait: 5 do
```

Comment:
Ah, fine. I'll remove the override in the example, then. It shouldn't matter in the meantime.